src/auth/auth-router.js

```diff
@@ -0,0 +1,44 @@
+const express = require('express');
+
+const manager = require('./auth-manager');
+const router = express.Router();
```

Comment:
Any reason?

Since this code is within the `auth` domain, the names `manager` and `router` are clear. However if the `auth-router` was referenced in the `user` domain, I would agree 100%.